src/auxiliary/math.jl

```diff
@@ -207,4 +207,70 @@ Return `x` if `x` is negative, else zero. In other words, return
 @inline function negative_part(x)
     return min(x, zero(x))
 end
+
+"""
+    stolarsky_mean(x, y)
```

Comment:
```suggestion
    stolarsky_mean(x, y, gamma)
```

Reply:
Whoops, this was a mistake on my part from the initial implementation I provided to Simon. Sorry about that